python/cudf/cudf/core/series.py

```diff
@@ -4602,6 +4602,21 @@ def strftime(self, date_format, *args, **kwargs):
             data=str_col, index=self.series._index, name=self.series.name
         )
 
+    @copy_docstring(DatetimeIndex.tz_localize)
```

Comment:
This might be weird, but I would expect to copy the docstring from Series to Index, rather than Index to Series. I view the Series as the more fundamental object, but I don't know what precedents exist here in either class.

Reply:
Sounds good. I agree with you!